src/api/oktapi.ts

```diff
@@ -41,8 +30,12 @@ export async function testApi(
   const response = await axios({
     method: 'GET',
```

Comment:
@SaiSrinivasPendyala  can't we use abstracted request function here?

Reply:
We are importing the variables for other purposes. So I'm just using them. And also this is a separate axios call.